nuget/lib/dependabot/nuget/file_fetcher.rb

```diff
@@ -102,9 +99,16 @@ def project_files
             project_files
           end
       rescue Octokit::NotFound, Gitlab::Error::NotFound
+        raise_dependency_file_not_found
+      end
+
+      sig { returns(T.noreturn) }
+      def raise_dependency_file_not_found
         raise(
-          Dependabot::DependencyFileNotFound,
-          File.join(directory, "<anything>.(cs|vb|fs)proj")
+          Dependabot::DependencyFileNotFound.new(
+            File.join(directory, "*.(sln|csproj|vbproj|fsproj|proj)"),
```

Comment:
any reason why we don't search for `.sln` files in the lookup above?

Reply:
Those are separately pulled in via `sln_file_names` on line 132.